repo2docker/app.py

```diff
@@ -642,7 +659,9 @@ def start(self):
                     raise e
                 sys.exit(1)
 
-        if self.repo_type == 'local':
+        # if the source is a directory we will keep using it, assuming that
```

Comment:
@betatim How do I determine whether the user wants to use the directory or not?

Reply:
Got it. Perhaps: remove the wording re: assumption.

Maybe `# If the source to be executed is a directory, continue using the directory. In the case of a local directory, it is used as both the source and target`

Reply:
Thanks!